riscv_ctg/env/arch_test.h

```diff
@@ -100,6 +100,28 @@
   #endif
 #endif
 
+#if ZFINX==1
+  #define FLREG lw
+  #define FSREG sw
```

Comment:
This is not correct. Even if only Zfinx is supported, the entire register needs to be stored. Hence the `FSREG` should be mapped to `SREG`. I am not sure of the loads. It is possibly fine for the loads to only load partial contents of the register.

Reply:
corrected...!